data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/config/GetSpec.java

```diff
@@ -53,12 +65,18 @@ public GetSpec(final DataProvider dataProvider, final JsonProvider jsonProvider)
         @OpenApiResponse(status = RES_INTERNAL_ERROR)
       })
   @Override
-  public void handle(final Context ctx) throws Exception {
+  public void handle(@NotNull final Context ctx) throws Exception {
+    adapt(ctx);
+  }
+
+  @Override
+  public void handleRequest(RestApiRequest request) throws JsonProcessingException {
     try {
-      ctx.json(jsonProvider.objectToJSON(configProvider.getConfig()));
+      request.respondOk(configProvider);
```

Comment:
Shouldn't need to give a value to `respondOk()` method, however the `SerializableTypeDefinition` requires a value type.